libraries/shared/src/Preferences.h

```diff
@@ -391,10 +393,21 @@ class CheckPreference : public BoolPreference {
         : BoolPreference(category, name, getter, setter) { }
     Type getType() override { return Checkbox; }
 
+    //bool getEnabled() { return _isEnabled; }
+    //void setEnabled(const bool enabled) { _isEnabled = enabled; }
+
```

Comment:
Why is this being added?

Reply:
> Why is this being added?

I was using that for a previous approach and decided to leave it as a comment rather than taking it out, since it might still be useful at some point. You can remove it if you want.